README.md

```diff
@@ -0,0 +1,77 @@
+<a name="title"></a>
```

Comment:
:red_circle: Please make sure your README file includes the following sections: Getting Started, Contributing, Show Your Support and Acknowledgments. The FAQ section is optional, and the Live Demo section can be skipped for this project.

Note: The only optional section is FAQ. and in some projects like this one you can skip the Live Demo section)

Reply:
### Hi @bienvenuushindi,

Thanks for your review and feedback. I will address your points but I have some questions though.

## Linters

The instructions are not clear for me in setting them up. I have created the folders path `.github/workflows` and included the file `linters.yml` inside that path. However, it did not work and I do not know why!
Perhaps, they should be created on the main branch instead of the feature branch?

## ReadMe

I have changed the ReadMe file to fit my needs and what I understood. 
I do not know what to do with acknowledgments or contributing for this project.
Also, I have included getting started and only renamed it to instructions.

Looking forward to your feedback,

Reply:
@Mahammad-Mostafa  Thanks for asking. 
To set up the linters properly for this project (HTML + CSS), please follow the steps in this [link](https://github.com/microverseinc/linters-config/tree/master/html-css). If you are still having challenges, please ask your coding/learning partner or one of your session mates (morning or standup) for assistance.

It is important to rely on your teammates when facing challenges. Your teammates are there to help you learn and grow, and they can provide valuable insights and support. If you are struggling with something, don't be afraid to ask for help. Your teammates will be happy to help you succeed.

Reply:
Hi @bienvenuushindi,

I have fixed the issues with my linters config and passed all checks successfully.
I request a code review for approval.

Looking forward to your feedback,

Reply:
@Mahammad-Mostafa Congratulations, I am grateful for your dedication and hard work.  :+1: :clap:   you should use the dashboard to request a review and not GitHub. The dashboard is a much more efficient way to request reviews, and it allows you to track the status of your requests.

Reply:
Thank you @bienvenuushindi for your support and approving my request 🤩
However, I requested review through GitHub because the dashboard does not help make a request 😕
This window appears when I choose request a review for the activity 👇🏻

![Screenshot 2023-06-01 at 12 02 57 AM](https://github.com/Mahammad-Mostafa/hello-microverse/assets/91811315/bc6c193c-5ae1-4e6d-b5f9-20b3f595cb2b)

Reply:
@Mahammad-Mostafa What happens when you click on the blue button??

Note: :red_square:  Please don't merge your pull request until you see an approved status in your dashboard for this project  :red_square:

Reply:
It does what the button exactly says **Rate Review**. It opens a form to rate your feedback and my notes on them.

Reply:
@Mahammad-Mostafa  Kindly rate first, then after submitting your rating, you will be redirected to another form where you can request for a re-review. It's a simple procedure, so please follow it.

Reply:
I understand now 😲 I was a little confused 😅 This means every review even not approved will require rating the reviewer 😉

I have rated and requested the review from the dashboard successfully 👌🏻

Thanks a lot @bienvenuushindi for your assistance 😃